### ISSUE TITLE
`make build` does not install sphinx in virtual environment

### ISSUE THREAD

**Issue by ramasesh** [google/uv-metrics#8]
To start editing some docs, I ran `make build` in the root directory to get the `env` folder setup, and then ran `sphinx-reload docs`.  I get the error 
```
/bin/sh: ../env/bin/sphinx-build: No such file or directory
```
Looking in the `env/bin` folder, there is no sphinx-build.  However, there is one in the Caliban `env/bin` folder, so somehow the setup is not happening in UV in the same way.

**Comment by ramasesh** [comment]
Issue is a simple line that needs to be added to the `Makefile`:

`$(PIP) install -r docs/requirements.txt`

**Comment by ramasesh** [comment]
Fixed in PR #9 , so closing.
